client/consensus/Cargo.toml

```diff
@@ -9,19 +9,21 @@ repository = "https://github.com/paritytech/frontier/"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"] }
-sp-core = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-blockchain = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-runtime = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-api = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sc-client-api = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-block-builder = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-inherents = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
+sp-core = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "rococo-v1" }
```

Comment:
I think we should keep the branch frontier in `master`.
One possibility is to have a `rococo-v1` branch on frontier, but the issue will be that someone will have to maintain it :/

Reply:
I truly don't see why we would have a Frontier branch anymore. It's just another dependency to manage and it's honestly just more that @sorpaas would have to authorise, since he would need to merge it in Substrate land (afaik nobody else is looking into these things).

I've had this PR up now for a while: https://github.com/paritytech/substrate/pull/8500

Reply:
I agree having less branch is probably better, but in that case we should target master, not a dedicated branch (like `rococo-v1`)

Reply:
Aren't most parachain teams working against `rococo-v1`?

Reply:
Happy to move it to master in any event.

Reply:
Most parachain are probably working on `rococo-v1` but there are non-parachain team also using frontier.